src/cdef.rs

```diff
@@ -191,9 +192,8 @@ fn adjust_strength(strength: i32, var: i32) -> i32 {
 // CDEF parameters are stored for each 64 by 64 block of pixels.
 // The CDEF filter is applied on each 8 by 8 block of pixels.
 // Reference: http://av1-spec.argondesign.com/av1-spec/av1-spec.html#cdef-process
-pub fn cdef_frame(fi: &FrameInvariants, rec: &mut Frame, bc: &mut BlockContext) {
-    let bit_depth = 8;
-    let coeff_shift = bit_depth - 8;
+pub fn cdef_frame(fi: &FrameInvariants, rec: &mut Frame, bc: &mut BlockContext, bit_depth: usize) {
```

Comment:
I'd fold it in the `FrameInvariants`.

Reply:
If you are suggesting to duplicate `bit_depth` between `Sequence` and `FrameInvariants`, I don't think that this is a good idea.

Reply:
We can wait with this until a future Sequence-inside-FrameInvariants patch.